src/test/java/core/basesyntax/service/impl/FileReaderServiceImplTest.java

```diff
@@ -0,0 +1,49 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FileReaderServiceImplTest {
+    private static final String INPUT_PATH = "src/test/java/resourcesTest/inputFileTest.csv";
+    private static final String EMPTY_FILE = "src/test/java/resourcesTest/empty.csv";

```

Comment:
It is better not to make these fields final. It is better to create them in the test or in beforeClass